data/src/main/java/com/ronnie/data/db/PixaBayRoomDb.kt

```diff
@@ -17,36 +18,11 @@ import javax.inject.Singleton
  */
 
 @Database(
-    entities = [Image::class, RemoteKey::class],
-    version = 5, exportSchema = false
+    entities = [ImagesEntity::class, RemoteKey::class],
+    version = 1, exportSchema = false
```

Comment:
You should be increasing the version not decreasing it.

Reply:
Noted will work on that